src/data/events/highlights.js

```diff
@@ -1,78 +1,47 @@
-	created Hackschool, our introductory web development workshop series, to give
-	you a place to start in this wonderful universe of possibilities. By the end,
-	we hope to get you excited about CS, equip you with basic tools to build your
-	own web applications, and provide a community to learn from, bond with, and
-	hack with.
+const hackFamDescription = `
+	It’s hard to make friends and stay in touch during quarantine, but ACM Hack is
+	here to make it better. HackFam is a family system for people to hang out,
+	study, go to ~virtual~ events, and make friends in :) With a family head and
+	your family members, you will be able to make it through pandemic with plenty
+	of smiles and happy memories.
 `;
 
-const learnpyDescription = `
-	Learn.py is a quarter-long Python course with weekly sessions. Our goal is to
-	make learning coding, specifically in Python, easy and accessible to everyone
-	and while introducing you to cool Python applications such as data science,
-	gaming, web dev, and more! No programming/Python background is required.
-`;
-
-const toolingseriesDescription = `
-	In Tooling Series, we teach you how to take 
-	advantage of various tools that may help you save time and improve your 
-	workflow. You'll learn about text editors, version control, the command-line, 
-	and more! The recommended level of experience will vary depending on the workshop.
-`;
-
-const passiontalksDescription = `
-	Ever wondered how you can break into tech?   
-	Spend some quality time at Passion Talks learning about mind-blowing applications
-	of tech in different industries from passionate UCLA students! Whether you’re 
-	south or north campus, a complete newbie or a coding genius, tech is for everyone, 
-	and we’re bringing it to you!
+const bruinQuestDescription = `
+	Ready for the ultimate quest of the school year? Get ready for Bruin Quest,
+	where you will get to explore a virtual UCLA campus to complete puzzles and
+	challenges to win prizes. On this adventure, you and a team will have to combine
+	your wits and creativity to try maximize your points and come out on top as the
+	Bruin Quest champion.
 `;
 
 const highlightedEvents = [
 	{
-		name: 'Hackschool',
-		description: hackschoolDescription,
-		imgFilePath: 'event/2019f-hackschool.png',
-		button: 'Curriculum',
-		link: 'https://github.com/uclaacm/hackschool-f19'
-	},
-	{
-		name: 'Hack on the Hill',
-		description: hothDescription,
-		imgFilePath: 'event/2019w-hoth-banner.png',
-		button: 'Learn More',
-		link: 'https://hoth.splashthat.com/'
-	},
-	{
-		name: 'Passion Talks',
-		description: passiontalksDescription,
-		imgFilePath: 'event/2020s-passiontalks.png',
-		button: 'Recordings',
-		link: 'https://www.youtube.com/watch?v=BB9uzqJgeMg&list=PLPO7_kXilXFartXh7vu7OI6zq8JOF5k5u&index=2'
-
+		name: 'Bloom: Hackschool',
+		description: bloomDescription,
+		imgFilePath: 'event/2020f-bloom-hackschool-banner.png', // TODO: change
```

Comment:
Ignore this `TODO` . I forgot to remove it before I pushed. 

I will remove it in the next PR to update events. I don't wanna waste extra netlify build time when just to remove this comment in a new commit.

Reply:
we are not that tight on netlify build budget haha